fdk-fulltext-search-api.yaml

```diff
@@ -112,6 +118,68 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/SearchResult'
+        '500':
+          description: Server error
+          content:
+            application/json:
+              schema: $ref '#/components/schemas/ErrorMessage'
+  /datasets:
+    post:
+      summary: Returns a list of hits from full text searches in Felles Datakatalog's datasets
+      operationId: searchDataSets
+      description: An ObjectArray containing relevant hits and metadata for words or sentences in FDK's content"
+      requestBody:
+        required: false
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/SearchRequest'
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SearchResult'
+        '500':
+          description: Server error
+          content:
+            application/json:
+              schema: $ref '#/components/schemas/ErrorMessage'
+  /suggestion/{type}:
+    get:
+      summary: Returns a list of auto complete suggestions for the specified content type in FDKs content. TODO-> Default to all content types, [dataservices,concepts,informationmodels]
```

Comment:
Eg har alltid stussa på at vi brukar uttrykket "auto complete" i back-end. Er auto complete ein back-end funksjon? Eg har oppfatta det slik at _front-end_ tilbyr auto complete ved å bruke eit _søk_ mot back-end.

Reply:
Enig i at autocomplete i backend blir feil, lette litt rundt for å finne et godt navn på endepunktet, suggestion er mye brukt og tenker også det forklarer hva ressursen gir ut